src/transformers/models/wav2vec2/processing_wav2vec2.py

```diff
@@ -59,7 +59,7 @@ def save_pretrained(self, save_directory):
 
         .. note::
 
-            This class method is simply calling :meth:`~transformers.PreTrainedFeatureExtractor.save_pretrained` and
+            This class method is simply calling :meth:`~transformers.SequenceFeatureExtractor.save_pretrained` and
```

Comment:
```suggestion
            This class method is simply calling :meth:`~transformers.FeatureExtractionSavingUtilsMixin.save_pretrained` and
```
sphinx won't resolve the reference since it's not documented on `SequenceFeatureExtractor`. Unless you add the documentation for `SequenceFeatureExtractor` manually in which case you should ignore this suggestion.

It's the same for the other references below.